bower.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "react-zeroclipboard-mixin",
+  "version": "0.1.2",
+  "description": "React mixin for automatic binding of the ZeroClipboard client to the clipboard action DOM elements",
+  "dependencies": {
+    "react": "0.14.3",
```

Comment:
I would prefer to use only `npm`, don't see the reason of including one more dependency management system